### PR TITLE
adds color triangle art

### PR DESCRIPTION
**Pull request by ingridclaire**
# Zero To Mastery - Canvas 2022 🎉🎉

👉 _**Please remove the below and replace with your own values, leaving the headers where they are.**_ 👈

## PR contains:
- colorful triangle art: number, placement and color of triangles changes on resize

## Breaking changes
None

## Screenshots
/Users/ingridcarlson/Documents/ZTM/open-source/canvas-2022/src/art/ingridclaire/icon.png

### CODE REVIEW

File: src/art/ingridclaire/script.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    const y3 = y2 + Math.floor(Math.random() * 50) + 50;
+
+    
+    ctx.beginPath();
+    ctx.moveTo(x1, y1);
+    ctx.lineTo(x2, y2);
+    ctx.lineTo(x3, y3);
+    ctx.fillStyle = chooseColor();
+    ctx.fill();
+
```
Comment (betiniakarandut) [inline src/art/ingridclaire/script.js:34]: Please endeavor to remove the extra space here.

File: src/art/ingridclaire/style.css [hunk 2]
```diff
@@ -0,0 +1,11 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+
```
Comment (betiniakarandut) [inline src/art/ingridclaire/style.css:6]: Please reduce it to single line spacing. You can leave line 4 blank and clean other empty lines (that is lines 5 ,6, and 7). It makes your code more neater. Thank you

File: src/art/ingridclaire/script.js (hunk 1 above)
Comment (betiniakarandut) [reply src/art/ingridclaire/script.js:34]: I mean remove the extra line (line 34 precisely).

**Review by betiniakarandut — CHANGES_REQUESTED** [review]
Remove extra spaces/lines.

File: src/art/ingridclaire/script.js (hunk 1 above)
Comment (ingridclaire) [reply src/art/ingridclaire/script.js]: thanks for the note - removed!

File: src/art/ingridclaire/style.css (hunk 2 above)
Comment (ingridclaire) [reply src/art/ingridclaire/style.css]: removed!

File: src/art/ingridclaire/index.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Spinning triangles</title>
+</head>
+
+<body>
+  <canvas id="canvas" width="500" height="500"></canvas>
+  <script src="script.js"></script>
+</body>
+
```
[the remark this answers is not in the document]
Comment (ingridclaire) [reply src/art/ingridclaire/index.html]: removed!

**Comment by sophiabrandt** [comment]
Thank you @ingridclaire.

**Pull request merged**